frame/authority-discovery/src/lib.rs

```diff
@@ -71,17 +71,24 @@ impl<T: Trait> pallet_session::OneSessionHandler<T::AccountId> for Module<T> {
 	where
 		I: Iterator<Item = (&'a T::AccountId, Self::Key)>,
 	{
-		let keys = authorities.map(|x| x.1).collect::<Vec<_>>();
+		let mut keys = authorities.map(|x| x.1).collect::<Vec<_>>();
+		keys.sort();
+		keys.dedup();
```

Comment:
probably harmless to keep but I don't think this is needed here.

Reply:
Actually it is def. not needed because of 
https://github.com/paritytech/substrate/blob/549050b7f1740c90855e777daf3f9700750ad7ff/frame/session/src/lib.rs#L447-L451

Reply:
Thanks for the link! Will remove these lines.